EIPS/eip-draft_Pull_Oracle_Interface.md

```diff
@@ -0,0 +1,122 @@
+---
+eip: xxxx
+title: Pull Oracle Interface
+author: Alliance of Decentralized Oracles (l.loya.b@gmail.com)
+discussions-to: https://github.com/ethereum/EIPs/issues/
+status: DRAFT
+type: Standards Track
+category: ERC
+created: 2019-11-06
+---
+
+## Simple Summary
+
+A standard interface for numeric pull oracles.
+
+## Abstract
+
+In order for ethereum smart contracts to interact with off-chain systems, oracles must be used. These oracles report values which are normally off-chain, allowing smart contracts to react to the state of off-chain systems. A standard interface for oracles is described here, allowing different oracle implementations to be interchangeable and lead to increased security for customers.  
+
+## Motivation
+
+As the value held by smart contracts that rely on off-chain data to execute grows, the security of oracles systems and the ability of these to be interchangeable and implemented in groups will grow in importance. 
+
+The Ethereum ecosystem currently has many different oracle implementations available, but they do not provide a unified interface. Smart contract systems would be locked into a single set of oracle implementations, or they would require developers to write adapters/ports specific to the oracle system chosen in a given project.
+
+
+## Background
+EIP 1154, for standardizing an oracle interface, was not widely adopted and abandoned as too early and specificaly too broad/general to provide any efficiency gains. This new proposal is a cooperation between current oracle systems and direct users of these. The specification below reflects this collaboration and addresses the main reason why 1154 was not successful.
```

Comment:
Please use `EIP-1154` as the name and make it a link to the EIP. Likewise with 1154.

Reply:
updated all instances with EIP-1154 with link